docs/application/native/guides/text-input/tts.md

```diff
@@ -354,9 +352,9 @@ To obtain the current state, the supported voice list, and the current voice:
 
 - Get the default voice using the `tts_get_default_voice()` function.
 
-    The TTS synthesizes the text using the default voice, if you do not set the language and the voice type as parameters of the `tts_add_text()` function.
+    The TTS synthesizes the text using the default voice, if you do not set the language and the voice type as parameters of the `tts_add_text()` function:
```

Comment:
Ref: It is best not to use "the" in front of tool names.
```suggestion
    TTS synthesizes the text using the default voice if you do not set the language and the voice type as parameters of the `tts_add_text()` function:
```

Reply:
@stom-hwang,
Please remove "The" at the beginning of the sentence. Also, the comma after "voice" is not needed here.
Perhaps you missed it on your last commit.

Reply:
@stom-hwang,
a gentle reminder to incorporate this change,
Please remove "The" at the beginning of the sentence. Also, the comma after "voice" is not needed here.
Perhaps you missed it on your last commit.
```suggestion
    TTS synthesizes the text using the default voice if you do not set the language and the voice type as parameters of the `tts_add_text()` function:
```

Reply:
I'm really sorry that I miss it and thank you for reminding me.
I've applied your suggestion right now.
Would you please check again?